Jenkinsfile

```diff
@@ -15,38 +22,95 @@ pipeline {
 		timestamps()
 
 		// Prevent concurrent builds
-		disableConcurrentBuilds();
+		disableConcurrentBuilds()
 	}
 	stages {
-		stage ('1.3.3 Build') {
+		stage ('Discover Version Number') {
+			steps {
+				script {
+					def pom = readMavenPom file: 'pom.xml'
+					def currentVersion = pom.getVersion()
+					echo 'Current Version: ' + currentVersion
+
+					VERSION = currentVersion.substring(0, currentVersion.indexOf('-SNAPSHOT'))
+					echo 'Release Version: ' + VERSION
+
+					def parts = VERSION.tokenize('-')
+					echo 'Parts: ' + parts
+
+					def index = parts[1].toInteger()
+					echo 'Index: ' + index
```

Comment:
What is index? Is it actually the release/version number?

Reply:
Yeah, will rename, will also remove the echos as they were just there for testing purposes